app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,48 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+h3 {
+  font-size: 22px;
+  color: #05c2d1;
+}
+
+h1 {
+  text-align: center;
+  color: #05c2d1;
+  text-shadow: 2px 2px 2px #000;
+  font-size: 45px;
+  font-weight: 400;
+  padding-left: 100px;
+}
+
+.filter-mentors {
+  font-size: 15px;
+  color: #7c8c8e;
+  text-shadow: 1px 1px 1px #05c2d1;
+}
+
+body {
+  font-family: 'Raleway', sans-serif;
+}
+
+nav {
+  background-color: grey;
+}
+
+.left-logo {
+  color: red;
+  text-shadow: 2px 2px 10px #000
+}
```

Comment:
I'm not sure how it will work but we might want to talk to about breaking our css out into their own files.